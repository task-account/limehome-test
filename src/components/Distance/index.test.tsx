import renderer from "react-test-renderer";
import { Distance } from ".";

jest.mock("@utils/hooks", () => ({
  withStyles: () => () => ({ useStyles: jest.fn() })
}));

describe(`Distance`, () => {
  it("render", () => {
    const tree = renderer.create(<Distance distance={3.2123} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
