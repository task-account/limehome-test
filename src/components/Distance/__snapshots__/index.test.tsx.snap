// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Distance render 1`] = `
<View>
  <Text />
  <View
    style={
      Object {
        "flex": undefined,
        "marginBottom": undefined,
        "marginHorizontal": 4,
        "marginLeft": undefined,
        "marginRight": undefined,
        "marginTop": undefined,
        "marginVertical": undefined,
      }
    }
  />
  <Text>
    3.2 km from city centre
  </Text>
</View>
`;
